### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by AnastasOdyssea**

### CODE REVIEW

**Comment by AnastasOdyssea** [comment]
Hi .
Your code looks very clean to me and was easy to understand it.

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 <head>
     <link href='./resources/css/index.css' rel='stylesheet' type='text/css'>
+    Why you use type='text/css'?
```
Comment (med4kat) [inline index.html:7]: Good qn! Looks like It's not required with the HTML5 spec. Here is a link. Will remove it. Thanks!

https://stackoverflow.com/questions/5409114/is-type-text-css-necessary-in-a-link-tag

File: index.html [hunk 2]
```diff
@@ -16,6 +17,7 @@
     <section>
         <header class='copies'>Copies</header>
         <img src='./resources/van_gogh_almond_blossoms.jpeg' height=250px;>
+        Why you define the property height on html?
```
Comment (med4kat) [inline index.html:20]: https://www.w3schools.com/tags/tag_img.asp

I also need to add alt attribute as you did!

**Comment by med4kat** [comment]
Hi Anastas,

I figured it out how to comment the code. Please see my replies to your qns above. 
Many thanks for your review. I will now close this pull request as there are no changes.

**Comment by AnastasOdyssea** [comment]
How did you comment the code?
Did you find any documentation how to do it?

**Comment by med4kat** [comment]
Nope, didn't find docs. Just try and error.

https://user-images.githubusercontent.com/31211139/164205562-3e8a50b6-8107-45b1-a1fe-f88578aeff96.mov

**Pull request closed without merging**